test-automation-school/src/main/java/test/automation/school/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,73 @@
+package test.automation.school.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Created by lina.pugo on 5/17/2017.
+ */
+public class SearchPage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @FindBy(xpath = "//input[@id='ptxt']")
+    public WebElement findSearchfield;
+
+    @FindBy(xpath = "//input[@value='Искать']")
+    public WebElement searchButton;
+
+    @FindBy(xpath = "//select[@name='search_region']")
+    public WebElement findSearchRegion;
+
+    @FindBy(xpath = "//select[@name='pr']")
```

Comment:
The value of "name" tag is most likely dynamic, meaning it might change. But this should be checked with developers if possible.